pgcli/pgcompleter.py

```diff
@@ -539,15 +548,11 @@ def get_function_matches(self, suggestion, word_before_cursor, alias=False):
         if suggestion.filter == 'for_from_clause':
             # Only suggest functions allowed in FROM clause
             filt = lambda f: not f.is_aggregate and not f.is_window
-            funcs = self.populate_functions(suggestion.schema, filt)
-            if alias:
-                funcs = [self.case(f) + '() ' + self.alias(f,
-                    suggestion.table_refs) for f in funcs]
-            else:
-                funcs = [self.case(f) + '()' for f in funcs]
+            funcs = [self._make_cand(f, alias, suggestion, True)
```

Comment:
The boolean arguments to _make_cand are a little confusing here, maybe explicitly name them as keywords, like
`self._make_cand(f, alias, suggestion, function=True)`